R/utils.R

```diff
@@ -334,3 +334,9 @@ map_stat <- function(stat) {
     stat = stat_lookup$STAT[which(stat_lookup$stat == stat)]
   )
 }
+
+capture_dots <- function(...) {
```

Comment:
Since `capture_dots()` doesn't really do any quasiquotation or metaprogramming, I think we should call this helper `flatten_dots()`.

Reply:
Resolved in 57ae862